pyriemann/clustering.py

```diff
@@ -305,14 +305,18 @@ class Potato(BaseEstimator, TransformerMixin, ClassifierMixin):
 
     References
     ----------
-    .. [1] A. Barachant, A. Andreev and M. Congedo, "The Riemannian Potato: an
-        automatic and adaptive artifact detection method for online experiments
-        using Riemannian geometry", in Proceedings of TOBI Workshop IV,
-        p. 19-20, 2013.
-
-    .. [2] Q. Barthélemy, L. Mayaud, D. Ojeda, M. Congedo, "The Riemannian
-        potato field: a tool for online signal quality index of EEG", IEEE
-        TNSRE, 2019.
+    .. [1] `The Riemannian Potato: an automatic and adaptive artifact detection
+    method for online experiments using Riemannian geometry
+    <https://hal.archives-ouvertes.fr/hal-00781701>`_
+    A. Barachant, A Andreev, and M. Congedo. TOBI Workshop lV, Jan 2013, Sion,
+    Switzerland. pp.19-20.
+
+    .. [2] `The Riemannian Potato Field: A Tool for Online Signal Quality Index
+    of EEG
+    <https://hal.archives-ouvertes.fr/hal-02015909>`_
+    Q. Barthélemy, L. Mayaud, D. Ojeda, and M. Congedo. IEEE Transactions
+    on Neural Systems and Rehabilitation Engineering, IEEE Institute of
+    Electrical and Electronics Engineers, 2019, 27 (2), pp.244-255
     """
```

Comment:
check your indentation. When you write .. in ReStructuredText the next line should be indented with block.